lib/matplotlib/axis.py

```diff
@@ -2090,7 +2090,7 @@ def _set_tick_locations(self, ticks, *, minor=False):
 
     def set_ticks(self, ticks, labels=None, *, minor=False, **kwargs):
         """
-        Set this Axis' tick locations and optionally labels.
+        Set this Axis' tick locations and optionally tick labels.
```

Comment:
The function is then `Axes.set_yticks`. While "y" is in the function name, it's nice to mention the axis in the docstring explicitly, because Axes has more than one axis.